about.html

```diff
@@ -0,0 +1,160 @@
+            unknown event now gathers all works of life with a turn up of thousands each year. For the first Time, its in AFRICA , Kampala, Uganda in E.Africa this OCTOBER</p>
+             <p class="desc2">Please contact @www.MunyonyoCommonwealth esort.ug, Along Lake Victoria Shores, @www.TripeAdvisors.com, www.Eaafricansafari.com</p>
+             
+             </div>
+             <div class="mid-label">
+              <div class="footer-head ">
+                <a href="#"> 
+                  <img class="logo-img" src="images/R.png" alt="logo"/></a>
+              </div>
+             <div class="footer-content">
+    
+              <h3>REMOTE <br>GLOBAL <br>SUMMIT</h3>
+              <p>
+                Ⓒ 2022 WFH Uganda <br />
+                All Rights Reserved
+              </p>
+            
+            
+            </div>
+          </div>
+
+             <div class="middle">
+              <h2 class="date">2022.10.15(MON ~ 17(WED)</h2>
+              <h3>REMOTE DEV's GLOBAL SUMMIT ~2022</h3>
+              <p>
+                Kampala, Uganda (The Pearl of AFRICA) <br />
+                Its has never been better
+              </p>
+            
+            
+            </div>
+          </footer>
+             <div class="past">
+                <h2 class="past-header">See the past Global Summits</h2> <hr>
+                <p class="desc">Take a look at the last two global summits which took place in Buenos Aires and in warsaw</p>
+               <div class="past-img">
+                
+                <img src="images/conf.jpg" alt="" class="pas"><div class="center">
+                  <h4>2020</h4><p>Remote Global summit 2020 warsaw</p></div>
+                  <img src="images/devConf.png" alt="" class="pas">
+                  <div class="center1">
+                    <h4>2021</h4><p>Remote Global summit 2021 Buenos Aires</p></div>
+              </div>
+             </div>
+             <div class="partners-sect">
+                <div class="partner-header"><p>Partners</p> <hr></div>
+                <div class="partners">
+                    <img src="images/google-1-1.svg" alt="">
+                    <img src="images/coinbase.svg" alt="">
+                    <img src="images/meta-1.svg" alt="">
+                    <img src="images/microsoft.svg" alt="">
+                   
+                    <img src="images/mozilla-1.svg" alt="">
+
+                </div>
+
+             </div>
+    </section>
+  </section>
+  <footer>
+    <div class="footer-head ">
+      <a href="#"> 
+        <img class="logo-img" src="images/R.png" alt="logo"/></a>
+    </div>
+  <div class="footer-content">
+    
+      <h3>REMOTE GLOBAL SUMMIT</h3>
+      <p>
+        Ⓒ 2022 WFH Uganda <br />
+        All Rights Reserved
+      </p>
+    
+    <P class="developer-infor">Developed By Paul MalongoK'la (U) @https://github.com/canon20paul </P>
+    </div>
+  </footer>
```

Comment:
- [ ] Your design needs improvement in the footer of the about page, right now it appears to display some elements blurred in the background. Please refer to the behance design and match it as much as possible to the behance design.

Behance Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/183242698-36e0aff9-2f08-42ff-aea3-9df1c5c894eb.png) |  ![image](https://user-images.githubusercontent.com/60618877/183242682-d20fd439-fd67-4d87-b88b-7e1a1b72c489.png)